adafruit_httpserver/request.py

```diff
@@ -33,8 +33,35 @@ def _add_field_value(self, field_name: str, value: Union[str, bytes]) -> None:
         else:
             self._storage[field_name].append(value)
 
-    def get(self, field_name: str, default: Any = None) -> Union[str, bytes, None]:
+    @staticmethod
+    def _html_output_encode(value):
```

Comment:
```suggestion
    def _encode_html_entities(value):
```

More verbose name in my opinion.

Reply:
changed with latest commit